tests/infer/mcmc/test_nuts.py

```diff
@@ -96,7 +96,6 @@ def jit_idfn(param):
     'fixture, num_samples, warmup_steps, expected_means, expected_precs, mean_tol, std_tol',
     TEST_CASES,
     ids=TEST_IDS)
-@pytest.mark.skip(reason='Slow test (https://github.com/pytorch/pytorch/issues/12190)')
```

Comment:
Same here - I think they can't be made much faster, and they will still add 5 mins to travis, so let us only keep these for local runs.